### PR TITLE
feature/53: added a react-native-popover-typescript

### PR DESCRIPTION
**Pull request by Gerasimov94**

### CODE REVIEW

File: src/assets/images/KebabMenu.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 /* tslint:disable */
 
 const KebabMenu: React.FC<any> = ({fill, width = 32, height = 32}) => (
 	<Svg width={width} height={height} viewBox='0 0 512 512'>
 		<G>
 			<G>
-				<Circle fill={fill} cx="256" cy="256" r="64"/>
-				<Circle fill={fill} cx="256" cy="448" r="64"/>
-				<Circle fill={fill} cx="256" cy="64" r="64"/>
+				<Ellipse fill={fill} x="256" y="256" rx="64" ry="64"/>
```
Comment (CodeAnnihilator) [inline src/assets/images/KebabMenu.tsx]: why G here?

File: src/screens/Rooms/components/RoomAddCard.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react';
+import {View, Text} from 'react-native';
+
+export default class RoomAddCard extends Component {
+	public render() {
+		return (
+			<View style={{flex: 1, justifyContent: 'center', alignItems: 'center'}}>
+				<Text>afasfasfas</Text>
```
Comment (CodeAnnihilator) [inline src/screens/Rooms/components/RoomAddCard.tsx]: what is this text for?

**Review by CodeAnnihilator — CHANGES_REQUESTED** [review]

File: src/screens/Rooms/components/RoomAddCard.tsx (hunk 2 above)
Comment (Gerasimov94) [reply src/screens/Rooms/components/RoomAddCard.tsx]: just for new screen, i work with it later at https://trello.com/c/ajslG1JE/31-upgrade-rooms-screen

File: src/assets/images/KebabMenu.tsx (hunk 1 above)
Comment (Gerasimov94) [reply src/assets/images/KebabMenu.tsx]: i leave it as is when i download a svg

Comment (CodeAnnihilator) [reply src/assets/images/KebabMenu.tsx]: G makes group of elements. Why double G?

**Review by CodeAnnihilator — APPROVED** [review]

**Pull request merged**